modules.json

```diff
@@ -210,9 +210,19 @@
             },
             "subworkflows": {
                 "nf-core": {
-                    "fasta_binning_concoct": {
```

Comment:
Was this subworkflow removed?
or was this overlooked by a TEMPLATE merge?

Reply:
It shouldn't be!?

Reply:
It's in the pipeline!

Reply:
Had to remove and re-install... strange...